Francisco_Trujillo/Asignments/dojos_ninjas/app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ActiveRecord::Base
+	has_many :messages, dependent: :destroy
+	belongs_to :blog
+	validates :title, :content, presence: true
+	validates :title, :content, length:{minimum: 7}
```

Comment:
You can actually combine these 2 into one line since you're validating the same fields with the same validations